src-ghc/Pact/Server/PactService.hs

```diff
@@ -54,11 +58,11 @@ initPactService CommandConfig {..} loggers = do
       mkSQLiteEnv logger True sqlc loggers >>= mkCEI
 
 
-applyCmd :: Logger -> Maybe EntityName -> PactDbEnv p -> MVar CommandState -> ExecutionMode -> Command a ->
+applyCmd :: Logger -> Maybe EntityName -> PactDbEnv p -> MVar CommandState -> GasEnv -> ExecutionMode -> Command a ->
             ProcessedCommand (PactRPC ParsedCode) -> IO CommandResult
-applyCmd _ _ _ _ ex cmd (ProcFail s) = return $ jsonResult ex (cmdToRequestKey cmd) s
-applyCmd logger conf dbv cv exMode _ (ProcSucc cmd) = do
-  r <- tryAny $ runCommand (CommandEnv conf exMode dbv cv logger) $ runPayload cmd
+applyCmd _ _ _ _ _ ex cmd (ProcFail s) = return $ jsonResult ex (cmdToRequestKey cmd) s
+applyCmd logger conf dbv cv gasEnv exMode _ (ProcSucc cmd) = do
+  r <- tryAny $ runCommand (CommandEnv conf exMode dbv cv logger) $ runPayload cmd gasEnv
```

Comment:
Please add gasEnv to `CommandEnv` so as to not have to pass it around

Reply:
Sure, updated.